products/firefox_iOS/review_iOSstrings.md

```diff
@@ -0,0 +1,75 @@
+
+For example: "Menu.NightModeAction.Title" is removed. If it's used in v5.x, and the iOS team plans to release an update to 5.0, localizations will miss that string.
+
+###Quickly review other locales
+Once you've checked "templates", you can pick at least one locale and see what changes:
+https://github.com/mozilla-l10n/firefoxios-l10n/pull/19/commits/a1c041a012e6996bd8bdaec88f21f3c34fe383b0
+
+##Reviewing strings during the cycle
+It's good to keep an eye on the strings landing during the Firefox for iOS cycle. In fact, if you wait to review strings when the PR is already opened, it's going to be complicated to get a fix in (until we resolve [Bug 1270855](https://bugzilla.mozilla.org/show_bug.cgi?id=1270855)).
+
+You should try to periodically check out the strings landing, and try to localize them in your head: how would you translate them? Would you be able to do it without the app? Is the localization comment clear enough?
+
+You might also need to identify the bug that added that string, see if there are screenshots, or ask if the iOS team can provide one. The point below explains how to find that bug.
+
+###How to find a bug
+Let's consider an example of new strings for Firefox for iOS, with [this past PR](https://github.com/mozilla-l10n/firefoxios-l10n/pull/19
+).
+
+The ID is 'NSCameraUsageDescription', the label 'This lets you take and upload photos.'.
+It doesn't have a localization comment, which is bad.
+
+So you open the main repository page:
+https://github.com/mozilla-mobile/firefox-ios
+
+And use the search box at the top, searching for the string ID (or the string):
+https://github.com/mozilla-mobile/firefox-ios/search?utf8=%E2%9C%93&q=NSCameraUsageDescription
+
+In this case you find two occurrences. The first one is the most interesting, so you open the file:
+https://github.com/mozilla-mobile/firefox-ios/blob/978bf46bb680291c61c5d21b6dc26472388a374f/Client/en.lproj/InfoPlist.strings
+
+Then use the "Blame" link on top:
+https://github.com/mozilla-mobile/firefox-ios/blame/978bf46bb680291c61c5d21b6dc26472388a374f/Client/en.lproj/InfoPlist.strings
+
+On the left, you should (almost always) find the bug reference:
+https://bugzilla.mozilla.org/show_bug.cgi?id=1312842
+
+#Wrapping up your work
+So you've reviewed the strings during the cycle, and the PR looks good? Wait! Don't merge it quite yet.
+
+You will need to announce this on [dev.l10n mailing list](https://lists.mozilla.org/listinfo/dev-l10n) as soon as you merge the PR. Prepare your email in advance. Here's [an example](https://groups.google.com/forum/#!searchin/mozilla.dev.l10n/ios$20v7%7Csort:relevance/mozilla.dev.l10n/oR5u3MdMLgE/952I4eyADAAJ) of what it should contain.
+
+[Pontoon](https://pontoon.mozilla.org/projects/firefox-for-ios/) will get the new strings automatically. Double check that [Pootle](https://mozilla.locamotion.org/projects/ios/) does too (it should - but better make extra sure).
```

Comment:
Maybe replace with "Make sure that tools like Pontoon and Pootle picked up the string changes automatically."